docs/02-sygma-sdk/05-Examples/01-NodeJS-Token-Transfer-Example.md

```diff
@@ -0,0 +1,345 @@
+
+:::info
+  The following example is written in `TypeScript`. You can use `CommonJS` with our SDK if you prefer to use standard `Node.js` for the script.
+:::
+
+#### Create folders for your project
+
+Create a new root directory for your project, e.g. `/sygma_test_project`. Using a code editor, navigate to the newly created folder in your terminal `cd sygma_test_project`. Install the Sygma SDK by following the instructions on [Installing The SDK](https://docs.buildwithsygma.com/sdk/quickstart/install).
+
+#### Initialize the project and install dependencies
+
+Initialize the project by running the following commands:
+
+```bash
+yarn init -y
+```
+
+```bash
+yarn add -D typescript dotenv ts-node
+```
+
+```bash
+yarn add ethers@5.7.2
+```
+
+Once initialized, ensure that the `package.json`, the lock files (`yarn.lock` or `package-lock.json`), and `node_modules` folders have been installed properly into the directory. 
+
+Create a `tsconfig.json` file in the root directory of your project and paste in the following code. This configuration file specifies the root files and the compiler options needed to compile the project. Specifically for this example, we will configure the target to ESNext, enable ES module loading, and set the output directory to dist.
+
+```json
+{
+	"compilerOptions": {
+		"composite": true,
+		"target": "ESNext",
+		"module": "CommonJS",
+		"allowJs": true,
+		"declaration": true,
+		"sourceMap": true,
+		"declarationMap": true,
+		"resolveJsonModule": true,
+		"skipLibCheck": true,
+		"strict": true,
+		"esModuleInterop": true,
+		"downlevelIteration": true,
+		"allowSyntheticDefaultImports": true,
+		"forceConsistentCasingInFileNames": true,
+		"moduleResolution": "node",
+        "outDir": "dist",
+        "baseUrl": "src"
+	},
+	"ts-node": {
+		"esm": true,
+		"experimentalSpecifierResolution": "node"
+	},
+	"include": [
+		"src"
+	]
+}
+```
+
+Next, create a `src` folder in the root directory of your project.
+Inside the `src` folder, create a `transfer.ts` file and an `index.ts` file. 
+
+Finally, because we are dealing with sensitive data such as a private key and an API key, we use environment variables to manage this data. Create a `.env` file in the root directory of your project. Paste in the following:
+
+```
+PROVIDER_API_KEY=your_provider_api_key_here
+PRIVATE_KEY=your_private_key_here
+```
+
+Copy and paste (in quotations) the `Goerli` API key obtained from Infura or Alchemy, as well as the exported private key from MetaMask.
+
+:::danger
+  Remember that your private key should **never** be exposed. Hence, **DO NOT** commit the `.env` file to GitHub. Exposing the private key could result in complete loss of your funds. **_For this example, please only paste in the exported private keys of a wallet that is designated for testing/development purposes_**.
```

Comment:
could be, do not commit your `.env` file to any repository and add it to your `.gitignore` file

Reply:
added